x-pack/elastic-agent/pkg/core/plugin/process/app.go

```diff
@@ -12,6 +12,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/elastic/beats/v7/x-pack/elastic-agent/pkg/agent/program"
```

Comment:
i guess goland just likes newlines

Reply:
yes it does, I spend to much time trying to re-order imports